opus/application/static_media/css/opus.css

```diff
@@ -889,8 +889,45 @@ a:-webkit-any-link, a::any-link {
 }
 
 #cart-sidebar {
-    padding-left: 3em;
-    padding-right: 3em;
+    padding-left: 0em;
+    padding-right: 1em;
+}
+
+/* Make both buttons the same width */
+#cart .op-cart-select-btn, #cart .op-cart-deselect-btn {
+    width: 4%;
+    outline: none;
+}
+
+/* Remove the focus/hover background for "v" & "x" icon in download options
+panel */
+#cart .op-cart-select-btn:hover, #cart .op-cart-deselect-btn:hover,
+#cart .op-cart-select-btn:focus, #cart .op-cart-deselect-btn:focus {
+    background-color: #F0F3F5 !important;
```

Comment:
Is this really the best way to do this? Isn't there some way to simply disable the hover action entirely? Setting the background to a hardcoded color seems like a hack.

Reply:
In our case, we don't want to disable the entire hover & focus effect. We only want to remove the outline and background color when we mouse over to "v" and "x". The background color of the hover effect is coming from button.bg-light:hover (from btn-light:hover if we change the class in html), and the easiest way to bypass this is to customize css for these two buttons. I've modified the styling to "transparent" (not a specific hardcoded color now) so the hover effect background color will just match to it's parent container which is still a .bg-light from table sub-header.